### PR TITLE
Add BC link for /bin

### PR DESCRIPTION
**Pull request by RobinMalfait**

### CODE REVIEW

**Comment by introwit** [comment]
@RobinMalfait a simple doubt, any specific reasons to use the facade in place of the `redirect()` helper? Don't exactly know the reason which to prefer when so can you help me with this?

File: app/Http/Controllers/HomeController.php [hunk 1]
```diff
@@ -13,4 +15,9 @@ public function rules()
     {
         return view('rules');
     }
+
+    public function redirectToPaste()
+    {
+        return Redirect::to('https://paste.laravel.io');
```
Comment (driesvints) [inline app/Http/Controllers/HomeController.php]: Can you change this to the redirect helper?

```php
return redirect()->away('https://paste.laravel.io');
```

File: app/Http/Controllers/HomeController.php [hunk 2]
```diff
@@ -13,4 +15,9 @@ public function rules()
     {
         return view('rules');
     }
+
+    public function redirectToPaste()
```
Comment (driesvints) [inline app/Http/Controllers/HomeController.php]: Rename this to `pastebin`.

File: app/Http/Controllers/HomeController.php (hunk 1 above)
Comment (RobinMalfait) [inline app/Http/Controllers/HomeController.php]: Sure! I was looking in other files and the first I found used `Validator::` Facade, so I used that one (/cc @introwit ). Let me update and adhere to your requests 👍

**Comment by driesvints** [comment]
Thanks! Pushed one more change. Realized the named route wasn't needed.

**Comment by RobinMalfait** [comment]
Gotcha, I was just following the rest of the document :) Thanks for merging!

**Pull request merged**